test/framework/easyconfigs/yeb/goolf-1.4.10.yeb

```diff
@@ -0,0 +1,39 @@
+_internal_variables_:
+    - &version 1.4.10
+    - &comp_name GCC
+    - &comp_version 4.7.2
+    - &comp [*comp_name, *comp_version]
+
+    - &blaslib OpenBLAS
+    - &blasver 0.2.6
+    - &blas !join [*blaslib, -, *blasver]
+    - &blas_suff -LAPACK-3.4.2
+
+    - &comp_mpi_tc_name gompi
```

Comment:
not used anywhere else, so just use `gompi` in the line below and drop this one?